mu-plugins/blocks/global-header-footer/postcss/_common.pcss

```diff
@@ -123,6 +123,17 @@ html {
 	& * {
 		box-sizing: border-box;
 	}
+
+	& .wp-block-navigation-item__content {
```

Comment:
Could this just be `a`? the double class parent selector should be enough specificity, and using just `a` would work for any link added in the future.